src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,214 @@
+                + "VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createCarStatement = connection
+                        .prepareStatement(insertRequest,
+                             Statement.RETURN_GENERATED_KEYS)) {
+            createCarStatement.setString(1, car.getModel());
+            createCarStatement.setLong(2, car.getManufacturer().getId());
+            createCarStatement.executeUpdate();
+            ResultSet resultSet = createCarStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't create a car = "
+                    + car, e);
+        }
+        insertDriversForCar(car);
+        return car;
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String selectRequest = "SELECT c.id, model, manufacturers_id, name, country "
+                + "FROM cars c JOIN manufacturers m ON c.manufacturers_id = m.id "
+                + "WHERE c.is_deleted = FALSE AND c.id = ?;";
+        Car car = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection
+                        .prepareStatement(selectRequest)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                car = parseCar(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get car by id = "
+                    + id + ". ", e);
+        }
+        if (car != null) {
+            car.setDriver(getDriversForCar(id));
+        }
+        return Optional.ofNullable(car);
+    }
+
+    @Override
+    public List<Car> getAll() {
+        String selectRequest = "SELECT c.id, model, manufacturers_id, name, country FROM cars c "
+                + "JOIN manufacturers m ON c.manufacturers_id = m.id "
+                + "WHERE c.is_deleted = FALSE;";
+        List<Car> cars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection
+                        .prepareStatement(selectRequest)) {
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                Car car = parseCar(resultSet);
+                cars.add(car);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get all cars!", e);
+        }
+        cars.forEach(c -> c.setDriver(getDriversForCar(c.getId())));
+        return cars;
+    }
+
+    @Override
+    public Car update(Car car) {
+        String updateRequest = "UPDATE cars SET model = ?, manufacturers_id = ? "
```

Comment:
```suggestion
        String query = "UPDATE cars SET model = ?, manufacturers_id = ? "
```
it is obvious from the context what this query does, use simpler naming